### ISSUE TITLE
Error when try to delete an element on the tree 

### ISSUE THREAD

**Issue by AlexSerrano22** [CarbonLDP/carbonldp-workbench#57]
This issue is related to #56 , if you reproduce this error and after that, you try to delete a document with the trash icon on the top you getting an error.
```
ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'state' of undefined
TypeError: Cannot read property 'state' of undefined
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.delete_node (0.63dc1e304e246417972e.chunk.js:1)
    at DocumentTreeViewComponent.emptyNode (0.63dc1e304e246417972e.chunk.js:1)
    at 0.63dc1e304e246417972e.chunk.js:1
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvoke (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:1)
    at e.jstree.plugins.wholerow.delete_node (0.63dc1e304e246417972e.chunk.js:1)
    at DocumentTreeViewComponent.emptyNode (0.63dc1e304e246417972e.chunk.js:1)
    at 0.63dc1e304e246417972e.chunk.js:1
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvoke (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at resolvePromise (polyfills.ca4447d0991a01fbd3bd.js:1)
    at resolvePromise (polyfills.ca4447d0991a01fbd3bd.js:1)
    at polyfills.ca4447d0991a01fbd3bd.js:1
    at ZoneDelegate.invokeTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvokeTask (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invokeTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Zone.runTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at drainMicroTaskQueue (polyfills.ca4447d0991a01fbd3bd.js:1)
    at ZoneTask.invokeTask [as invoke] (polyfills.ca4447d0991a01fbd3bd.js:1)
    at p (polyfills.ca4447d0991a01fbd3bd.js:1)
defaultErrorLogger @ app.43679dca33355c574d85.js:1
```

<img width="1137" alt="screen shot 2018-01-17 at 10 20 51 pm" src="https://user-images.githubusercontent.com/5456395/35080878-09727026-fbd6-11e7-82cf-0a319ff91e77.png">

**Comment by alvaro9210** [comment]
This is expected since the Tree library failed. By solving the first issue, this error won't happen again. Nevertheless, we need to handle these type of cases for uncontrolled situations.

**Comment by alvaro9210** [comment]
This has been solved in the `v1.0.0-alpha.3` of the workbench. Please update.
